### PR TITLE
Log error

### PR DESCRIPTION
**Pull request by arahamad**
@narkarum , Please review and approve accordingly

### CODE REVIEW

File: volume-providers/vpc/provider/get_volume.go [hunk 1]
```diff
@@ -15,40 +15,44 @@ import (
...
-	defer vpcs.Logger.Info("Exit GetVolume", zap.Reflect("VolumeID", id))
+func (vpcs *VPCSession) GetVolume(id string) (respVolume *provider.Volume, err error) {
+	vpcs.Logger.Debug("Entry of GetVolume method...")
+	defer vpcs.Logger.Debug("Exit from GetVolume method...")
 
-	var err error
-	var volume *models.Volume
+	vpcs.Logger.Info("Basic validation for volume ID...", zap.Reflect("VolumeID", id))
+	// validating volume ID
+	err = validateVolumeID(id)
```
[the remark this answers is not in the document]
Comment (arahamad) [reply volume-providers/vpc/provider/get_volume.go]: its in return `GetVolume(id string) (respVolume *provider.Volume, err error) `

File: volume-providers/vpc/provider/util.go [hunk 2]
```diff
@@ -29,6 +29,8 @@ var maxRetryGap = 30
 // retryGap ...
 var retryGap = 5
 
+var volumeTokenNumber = 5
```
[the remark this answers is not in the document]
Comment (arahamad) [reply volume-providers/vpc/provider/util.go]: sure, I can change

File: volume-providers/vpc/provider/delete_volume.go [hunk 3]
```diff
@@ -17,25 +17,40 @@ import (
...
-	vpcs.Logger.Info("Entry DeleteVolume", zap.Reflect("vol", vol))
-	defer vpcs.Logger.Info("Exit DeleteVolume", zap.Reflect("vol", vol))
+func (vpcs *VPCSession) DeleteVolume(volume *provider.Volume) (err error) {
+	vpcs.Logger.Debug("Entry of DeleteVolume method...")
+	defer vpcs.Logger.Debug("Exit from DeleteVolume method...")
 
-	var err error
-	_, err = vpcs.GetVolume(vol.VolumeID)
+	vpcs.Logger.Info("Validating basic inputs for DeleteVolume method...", zap.Reflect("VolumeDetails", volume))
+	err = validateVolume(volume)
```
[the remark this answers is not in the document]
Comment (narkarum) [reply volume-providers/vpc/provider/delete_volume.go]: Oh OK

**Comment by narkarum** [comment]
LGTM

**Pull request merged**